src/js/wrapper.ts

```diff
@@ -84,6 +83,9 @@ export const NATIVE: SentryNativeWrapper = {
 
     const event = this._processLevels(_event);
 
+    // Delete this metadata as this should not be sent to Sentry.
+    delete event.sdkProcessingMetadata;
```

Comment:
is this a new thing?

Reply:
Yup - https://github.com/getsentry/sentry-javascript/blob/00d359d2defdb730829b700d5c324a2f9cef553d/packages/core/src/request.ts#L88